src/docfx.website.themes/default/styles/docfx.css

```diff
@@ -960,3 +997,17 @@ div.embeddedvideo iframe {
 
 .mainContainer[dir='rtl'] main ul[role="tablist"] {
   margin: 0; }
+
+/* Color theme */
+
+/* These are not important, tune down **/
+.decalaration, .fieldValue, .parameters, .returns {
+  color: #a2a2a2;
+}
+
+/* Major sections, increase visibility **/
+#fields, #properties, #methods, #events {
+    color: #337ac4;
```

Comment:
This color is almost same with `#337ab7`, which is used in links in other places. This may mislead the user to try clicking on it. What about keep it black?

Reply:
Right, because I committed just a subset of the full `main.css` we use, see _e.g._ [this page](https://microsoft.github.io/MixedReality-WebRTC/api/Microsoft.MixedReality.WebRTC.PeerConnection.html) using the proposed style. Here we're missing the header override we use, which takes precedence over this color (which should therefore be removed):

```css
/** Content **/

#_content h1 {
  color: rgb(68, 68, 68);
  font-size: 40px;
  font-weight: 100;
  margin-top: 0;
}

#_content h2 {
  color: rgb(68, 68, 68);
  font-size: 30px;
  font-weight: 100;
  padding-top: 70px;
  margin-top: -50px;
}

#_content h3 {
  color: rgb(68, 68, 68);
  font-size: 22px;
  font-weight: 100;
  padding-top: 70px;
  margin-top: -50px;
}
```

Reply:
Thank you for the explanation. Would you remove this line of color change in this PR?